admin_guide/scheduling/descheduler.adoc

```diff
@@ -311,7 +316,7 @@ spec:
     spec:
         containers:
         - name: descheduler
-          image: descheduler
+          image: registry-proxy.engineering.redhat.com/rh-osbs/openshift3-ose-descheduler:v3.11.487-1.git.d435537
```

Comment:
Better an image that's outdated rather than an image that can not be pulled. Still, can you mention to check for the latest before deploying the job?

Reply:
Sorry. Took you literally in the BZ. I updated to `image: registry.access.redhat.com/openshift3/ose-descheduler`. Is that correct?

Reply:
`docker inspect registry.access.redhat.com/openshift3/ose-descheduler` says (among other bits):
```
       "Created": "2021-07-01T21:08:39.605372657Z",
        "Config": {
            "Env": [
                ...
                "BUILD_RELEASE=1.git.d435537",
                "BUILD_VERSION=v3.11.465",
```
It might still be valid as the latest image is not yet released for consumption. Let's go with it.

Reply:
I used `image: registry.access.redhat.com/openshift3/ose-descheduler` in my job file and it picked up that image: `registry.access.redhat.com/openshift3/ose-descheduler:v3.11.465-1.git.d435537`